ios/build.sh

```diff
@@ -1,13 +1,29 @@
 #!/usr/bin/env bash
 
-set -e
-# set -x
-# set -u <-- difficult to test if variable is set this way...
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*}/../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
 
-./kmbuild.sh $*
+# This script runs from its own folder
+cd "$(dirname "$THIS_SCRIPT")"
 
-if [ ! -z "$RELEASE_OEM" ]; then
-  pushd ../oem/firstvoices/ios > /dev/null
-  ./build.sh -lib-nobuild $*
-  popd > /dev/null
-fi
+# Include our resource functions; they're pretty useful!
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+
+# Please note that this build script (understandably) assumes that it is running on Mac OS X.
+verify_on_mac
+
+builder_describe "Builds Keyman Engine and the Keyman app for use on iOS devices - iPhone and iPad." \
+  "clean" \
+  "configure" \
+  "build" \
+  ":engine          Builds KeymanEngine.xcframework, usable by our main app and by third-party apps" \
+  ":app=keyman      Builds the Keyman app for iOS platforms" \
+  "--debug+         Avoids codesigning and adds full sourcemaps for the embedded predictive-text engine" \
+  "--sim-artifact+  Also outputs a simulator-friendly test artifact corresponding to the build"
```

Comment:
I'm not sure if we want these as default, given `--debug` is not default on any other builds. I understand where you are going on this though -- the release build won't work because of codesign by default. Hmm. Tricky.

Reply:
https://github.com/keymanapp/keyman/blob/a9e6c86740057eb1cc94b058a34dc54266063405/resources/builder.inc.sh#L757-L771

Options with a `+` propagate to child scripts, which is the use-case here.